src/Events.php

```diff
@@ -156,6 +157,13 @@ protected function possibly_add_notes() {
 		UpdateStoreDetails::possibly_add_note();
 	}
 
+	/**
+	 * Deletes notes that should be deleted.
+	 */
+	protected function possibly_delete_notes() {
+		NavigationNudge::delete_if_not_supported();
```

Comment:
We also have the `NavigationFeedback` note that could benefit from removal logic while we're updating this.

Reply:
Is two notes enough to justify putting a version of this method in `NoteTraits`?

Reply:
Definitely!  I think this would be a useful function.